addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -177,6 +177,40 @@ const MoveArticleToDialog = Dialog.extend({
     },
 });
 
+const ConfirmMoveDialog = Dialog.extend({
```

Comment:
If I understood well the goal of this dialog (set Private or Workspace in bold) this component does not worth it. We should keep what have been done and live with it.
And Luc is maybe sad but agrees this is not a must have (and we can live without a bold category in the confirm message)

Reply:
Ok, let's remove it.